lib/services/manager_api.dart

```diff
@@ -82,6 +94,7 @@ class ManagerAPI {
     }
     await _revancedAPI.clearAllCache();
     await _prefs.setString('apiUrl', url);
+    _toast.showBottom('settingsView.restartAppForChanges');
```

Comment:
I was wondering, if restarting is actually necessary?

Reply:
Is there an alt api I can use to test?

Reply:
Because it is hard to tell since manager caches all data.

Reply:
You can technically deploy the API locally. Alternatively the endpoint `deimos.revanced.app` exists.

Reply:
I can confirm that reloading is necessary, most likely because of the `dio` initializing in `revanced_api.dart`. Trying to re-initialize gives the error about dio already being initialized.